multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -157,6 +198,13 @@ write_files:
           "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
       done
 
+  - path: /opt/bin/host-rkt
```

Comment:
https://github.com/coreos/rkt/issues/2878 Would adding this to the commit message sufficient?